Sources/Kitura/CodableRouter.swift

```diff
@@ -408,6 +470,46 @@ extension Router {
         }
     }
 
+    // Get w/Optional Query Parameters
+    fileprivate func getSafely<Q: QueryParams, O: Codable>(_ route: String, handler: @escaping (Q?, @escaping CodableArrayResultClosure<O>) -> Void) {
+        get(route) { request, response, next in
```

Comment:
Now that #1281 is merged, this needs a call to `registerGetRoute()` at the start of the function.